sdk/samples/iot/README.md

```diff
@@ -208,90 +212,63 @@ Next you must create and connect an authenticated device. You can authenticate i
 
 This approach must be used for the following samples: `paho_iot_hub_c2d_sample`, `paho_iot_hub_methods_sample`, `paho_iot_hub_telemetry_sample`, `paho_iot_hub_twin_sample`, `paho_iot_hub_pnp_sample`, `paho_iot_hub_pnp_component_sample`, `paho_iot_provisioning_sample`
 
-<details><summary><i>Instructions to create a device using X.509 Self-Signed Certificate Authentication:</i></summary>
-<p>
-
-1. Generate a certificate
-
-   As a convenience, we provide a series of commands below for you to create a temporary certificate in order to run the samples. These certificates expire after 30 days and are provided ONLY to help you easily understand CA Certificates. When productizing against CA Certificates, you will need to use your own security best practices for certificate creation and lifetime management.
-
-    **WARNING: Certificates created by these commands MUST NOT be used in production-level code.**
+##### Generate a certificate
```

Comment:
I think you could just take this line out completely. I don't think it needs its own link. Otherwise looks good.

Reply:
good point 👍